index.html

```diff
@@ -0,0 +1,103 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style/IdexStyles.css">
+    <link rel="stylesheet" href="./style/responsive.css">
+    <title>Document</title>
+</head>
+<body class="grand_container__style">
+    <header class="header_container__style">
+        <div class="header_content__style">
+            <svg class="header_logo__style" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="35" height="37" viewBox="0 0 35 37"><g><g><image width="35" height="37" xlink:href="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAACMAAAAlCAYAAADIgFBEAAAGZklEQVRYR62Ye4wV1R3Hv985c3dn5sK2KIYaLTU2pkEE0RKKjygktS0+2+r6iNWgIAVDadFgG10fDdG0Nm1EjQr4aKrYKqmoqImPqhgfpWoUH9TaNippmxofKMudmd17Zr7mXGbx7rK7dy5y/roz9/f7nc/8zjm/xyHaGEmiiVLeLWoyiS4AHoAcgAGQFc+S8CnFN0lvbRhyS9kpWEYwjutHweNPIewjYAvIv3rKN1vrbx47lh8M2Ojt1T6+byfl9CZDmklgIogPkGtFFFWeazXXqDCSxiZ92R0Q9gf0VJ75140Zw/dbGR34f/t2TfCM/RnA2SD+E3aa80luG0l/RJhaao+ncDOADWFgFpBMy0IMlZMUJGm2CsCxIhZVA/+R4WwNC1NLs0WUlkNYFEX+2t2FGKoXx7YbxC0ie6qBcR86aOwCkyR2Xg78ztBMDwL+c0+BDNhJUx2UK3uZwNIw9G9rtj8IJknqswGuI803g4D/3tMgTUBfz5G9ROmUMKw8M/B+J8zH0peCNHszp+aNCSqPtgMi6R4AN5B8tqze9rT+XU9cnQZmyl7kp05vJ0yc2DUEesPQX1jWoJOTdCQAd2xvJnlhO7pJYlcKGBOF/tk7YWq1/unw+Ico8KeR7G/HoKQnnVcAnAPgOpI73d7KjqRKnNpNyHVutdrxUsMzcWLvIfhGGJrlrQw0/y9pPoAfkZwlaQGAbpLHtWMjSbLLBR0Shf4Z3BGYsrVhYE4g2duOIUkvA1hG0nnHLdmLAHpIlt5zjcCaZg/nmelmHGcXiZpaDf25bYIsATCH5JwBPUnnAphP8ph2bNUS+3uKrzFO7J0gN0aBubGsAUkuMb4C4MckXxiydBsB/IrkurL24jRbDOlbjGN7P6Bro6jyfFllSb8AcDTJE4fqSDoNwFKSR5W110jE4DLGqb3P9psFXV38sIyyyzMAnDfOI/nqcDqSNgBYSfLuMja3bdN4vyNb5WDWRYH/gzJKxSb9JYBJJE8fSUfS8QCuIjmjrF3H0RaMpCoAtydOJfmP0SYq4s/dJG8tA7Q7MNcC2JekC3CjDkmzAfyW5OGtZBuxrvDMfX2JOX/cOH7S4kv3BuByz/dIvldmAkmPA1hP8vrR5Ldu1Zc7w+z24jTl10RRx99awDiDVZLzyoAU+2umy1kADiepkfTiuH8G4F3q4swakRuqgXGV2LBD0r4AXFT9Dsn/l4UpgB4A8DzJX4+kV0uzBZRmMUmyywAdEIb+BaPAuCIoJvmTdkAKmGkA7iy8Ux9OP0nsaoDvME11YK7sjjAwxw2XsSV9DcDDAI4h+XG7MAWQK13/TvKKofqSOpI0e4wwc3dk7dSuA/hUFJhdNpqkuwD8l+TPdwekgJkEwAEdMTQZx2m2BNCsKPB/2IBJkvqxknd9FJlDmyeU9A0ALsfMILl9d2EKoDXFR13SbCeOs01kviQMKxs+r/Riu17UpmpY6RkQlnRv4d4rvwhIAXMAgAcBzCb5kXtXS+rLKU6JIv/77rkJRvuB2avK8zmu6pI0HcBqADNJ9n1RmALIndic5MLGcab3kHIzrVrl/wbBNPZObLsF3BCF5iAAbv+8NdqRbBdQ0lcArLEWPfW6vR3glVHkO+83xi59Uxxni+lpWUfFPGNM67DfLlCWaUl/PVuhHKdFkf/nZv1hO8r+ev62tflXPZqpe7KRaxT+9G6qVLzJHRXPJd1BYxeYIsEtTNPs0VzoAfXHali5rF0PDJWvJfWrIc6FhwuiTuNO6VSS543qGUl/AfAnkqt7e3WwMdmNbjFF3lsNzC3tQtXSbCGl0yHAWnNhVxffkhQVIWMx+XkL3fCMpLMAuHTgMvdHJAelhlrNnpRT8z3SUHoj9/AErP+6ux6RdDKAi4ua+KLeXozzOuxhXo5vizwklzJPvLVa9dcPiWErAbgj7Zq/qxobWNIUAK81CbqYsgJAZ/GuTnKr+93XZ08FeLLIidbmNAaVoMO4jrIx6ln+r/4+veNXvAqlLYAe7OzcsUkljQNQKUQPBuDKC794doX9KgfjCufmHnkzgLeHWQ53TeaClctPEyS4nDWVxPgBWQGC9DTJdwG4S6W9ALg6yHUTzWOCSw1NL64guXxgmX7j2g4ArqZxm6pV6Hf3eH3uAkmSq4mXAtgE4IwC1nnV3feNNFxt41bA1UauKzmT5CefAb7O7lXSYHbxAAAAAElFTkSuQmCC"/></g></g></svg>
```

Comment:
As soon as this sprite is quite big, better to have it separate from HTML, like so in `img` folder or even in sprite